fs/span-manager/span_manager.go

```diff
@@ -126,6 +129,10 @@ func New(ztoc *ztoc.Ztoc, r *io.SectionReader, cache cache.BlobCache, cacheOpt .
 		r:        r,
 		spans:    spans,
 		ztoc:     ztoc,
+		retries:  retries,
+	}
+	if m.retries == 0 {
+		m.retries = defaultRetries
```

Comment:
0 seems like a dangerous sentinel value here. Someone might specify 0 when they actually want 0. -1 may be safer. This might also feel weird to golang devs who expect defaults to be 0 and 0s to be defaults, though. Fixing this is not a blocker, but a response/explanation before merge would be appreciated.

Reply:
Fair enough. I've renamed it to maxSpanVerificationTries, which now defaults to 1 + (number of retries requested). It will only default to `defaultRetries` if it's still less than or equal to 0.